### ISSUE TITLE
jpegtran-bin@3.1.0 postinstall: `node lib/install.js` Exit status 1 on Windows 10 npm

### ISSUE THREAD

**Issue by managombo** [imagemin/jpegtran-bin#75]
Hello,
Please I have an issu with jpegtran installation on windows 10:

```
npm install --save jpegtran-bin

> jpegtran-bin@3.1.0 postinstall C:\Users\..\node_modules\jpegtran-bin
> node lib/install.js

> internal/child_process.js:298

    throw errnoException(err, 'spawn');
    ^

Error: spawn EPERM
    at exports._errnoException (util.js:873:11)
    at ChildProcess.spawn (internal/child_process.js:298:11)
    at exports.spawn (child_process.js:362:9)
    at C:\Users\hihihi\Documents\MEGA\Web-mobile-dev\mooc-webdev\work-angularjs\conFusion\node_modules\bin-check\index.js:22:12
    at C:\Users\hihihi\Documents\MEGA\Web-mobile-dev\mooc-webdev\work-angularjs\conFusion\node_modules\executable\index.js:27:4
    at FSReqWrap.oncomplete (fs.js:82:15)
 [                ..] / postinstall: info lifecycle jpegtran-bin@3.1.0~postinstall: Failed to exec postinstall scriptnpm WARN optional Skipping failed optional dependency /chokidar/fsevents:
npm WARN notsup Not compatible with your operating system or architecture: fsevents@1.0.14
npm ERR! Windows_NT 10.0.10586
npm ERR! argv "C:\\Program Files\\nodejs\\node.exe" "C:\\Users\\...\\node_modules\\npm\\bin\\npm-cli.js" "install" "--save" "jpegtran-bin"
npm ERR! node v4.4.7
npm ERR! npm  v3.10.6
npm ERR! code ELIFECYCLE

npm ERR! jpegtran-bin@3.1.0 postinstall: `node lib/install.js`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the jpegtran-bin@3.1.0 postinstall script 'node lib/install.js'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the jpegtran-bin package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node lib/install.js
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs jpegtran-bin
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls jpegtran-bin
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     C:\Users\..\npm-debug.log
```

Please help me to resolv the situation. Thank you.

**Comment by managombo** [comment]
Here is the log file.
[npm-debug.log.txt](https://github.com/imagemin/jpegtran-bin/files/428749/npm-debug.log.txt)

**Comment by majabee** [comment]
I have the same problem on Mac os 10.11.6

**Comment by mfilotto** [comment]
Same for me, it's probably because I'm behind a corporate proxy and install.js is sourcing 'https://downloads.sourceforge.net/project/libjpeg-turbo/1.5.1/libjpeg-turbo-1.5.1.tar.gz' directly

**Comment by 1000ch** [comment]
Might be solved by bundling source (#96).
